README.md

```diff
@@ -11,6 +11,7 @@ Java idiomatic client for [Cloud Logging][product-docs].
 ## Quickstart
 
 If you are using Maven with [BOM][libraries-bom], add this to your pom.xml file
+
```

Comment:
This file is autogenerated. See here: https://github.com/googleapis/java-logging/blob/master/.readme-partials.yaml where you need to make these changes so they are not overridden

Reply:
I will update both places :+1:

Reply:
I've found that an update to the first part of the README should also be applied to [java_library/README.md](https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/java_library/README.md) template. I will do it in a separate PR.